### PR TITLE
The second function of zapper (asking area zapper condition)

### PR DESCRIPTION
**Pull request by winonecheng**
- Asking area zapper condition:
    1. send confirm template
    2. get the web of zapper map screenshot and upload to imgur.com if user click comfirm
    3. send img if second step success
 
- To do:
    1. Update readme (install Xvfb and chrome driver and add variables in setting files)
    2. Improve the process of sending zapper map img because the waiting time is so long
    3. Modify the ZAPPER_MAP_URL after the web is online

### CODE REVIEW

File: denguefever_tw/dengue_linebot/utils.py [hunk 1]
```diff
@@ -1,18 +1,26 @@
+from django.conf import settings
+
 import logging
+import requests
+from time import sleep
+
+from selenium import webdriver
+from pyvirtualdisplay import Display
```
Comment (Lee-W) [inline denguefever_tw/dengue_linebot/utils.py]: There is no need to add this blank line.

**Review by Lee-W — COMMENTED** [review]
It would be better if you also add "chrome" as prerequisite in README.md

File: denguefever_tw/dengue_linebot/utils.py (hunk 1 above)
Comment (winonecheng) [reply denguefever_tw/dengue_linebot/utils.py]: Fixed

**Review by Lee-W — APPROVED** [review]
LGTM. Will merge.

**Pull request merged**